addon/components/lt-column-resizer.js

```diff
@@ -1,44 +1,46 @@
+import classic from 'ember-classic-decorator';
+import { classNameBindings } from '@ember-decorators/component';
 import Component from '@ember/component';
 import closest from 'ember-light-table/utils/closest';
 
 const TOP_LEVEL_CLASS = '.ember-light-table';
 
-export default Component.extend({
-  classNameBindings: [':lt-column-resizer', 'isResizing'],
-  column: null,
-  resizeOnDrag: false,
-
-  isResizing: false,
-  startWidth: null,
-  startX: null,
+@classic
+@classNameBindings(':lt-column-resizer', 'isResizing')
+export default class LtColumnResizer extends Component {
+  column = null;
+  resizeOnDrag = false;
+  isResizing = false;
```

Comment:
Guessing this should be tracked? Anything we want to update ever should be tracked.

Reply:
Complete